codalab/rest/interpret.py

```diff
@@ -608,5 +610,25 @@ def expand_raw_item(raw_item):
                 raw_items.append(subworksheet_item(worksheet_info) + (None, None))
 
         return raw_items
+    else:
+        raise Exception("not a search query")
```

Comment:
It seems a bit dangerous and confusing that this method throws a method but line 177 doesn't try to catch it. Would it be better to just return a empty array if we don't expect it to fail. If we want to enforce that the input must be a search query, better to document it in the function comment block